scripts/vaccination_alerts/utils.ts

```diff
@@ -23,14 +24,13 @@ export function getCmsVaccinationInfo(): RegionVaccinePhaseInfoMap {
     .value();
 }
 
-// TODO: Fetch this from Firebase, return an empty object if Firebase
-// doesn't have any data.
-export function getFirebaseVaccinationInfo(): RegionVaccineVersionMap {
-  return {
-    // '29': {
-    //   emailAlertVersion: 0,
-    // },
-  };
+export async function getFirebaseVaccinationInfo(): Promise<
+  RegionVaccineVersionMap
+> {
+  const db = await getFirestore();
+  const versionDoc = await db.doc('info/vaccinationInfoUpdates').get();
```

Comment:
Oh, hrm...  I didn't realize we'd store this as a single document.  I was imagining we'd have a collection with a single document for each fips code...  But storing it as a single document is probably easier and should be fine as long as it doesn't exceed 1MB, which it probably won't unless we start adding additional per-fips data to it...  So we can do this, at least for now!

Reply:
Let's change it now then so we don't have to worry about that limitation, I will take than into account for the `vaccine-alerts` collection too 👍

Reply:
Updated the Firebase structure, let me know if it makes sense!